tests/test_obsidian_preflight.py

```
@@ -89,6 +89,38 @@ def test_require_cli_missing_binary_returns_3(self):
+                return True
+
+            def help(self) -> CommandResult:
+                return CommandResult(ok=True, stdout="", stderr="", returncode=0)
+
+        with tempfile.TemporaryDirectory() as tmp:
+            skills_root = Path(tmp)
+            _create_required_skills(skills_root)
+            stdout = io.StringIO()
+
+            with patch("shared.scripts.preflight_obsidian.ObsidianAdapter", FakeObsidianAdapter):
+                with contextlib.redirect_stdout(stdout):
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)